### ISSUE TITLE
Name tag missing in PlaylistItem

### ISSUE THREAD

**Issue by Angus71** [hbenl/mpc-js-core#5]
Hi Holger,

While using the playlist related functions, I found that the `Name` tag is missing in the `PlaylistInfo` implementation.

Wouldn't be big of a problem, but I was using it in a previous project and after switching to your package it's gone ... 😄 

I've added a 'simple' patch, it worked for me

```patch
diff --git a/src/objects/playlists.ts b/src/objects/playlists.ts
index 6240053..5d8562a 100644
--- a/src/objects/playlists.ts
+++ b/src/objects/playlists.ts
@@ -13,7 +13,8 @@ export class PlaylistItem {
 
 	id: number;
 	position: number;	
-	title: string;
+    title: string;
+    name: string;
 	artist: string;
 	album: string;
 	albumArtist: string;
@@ -27,6 +28,7 @@ export class PlaylistItem {
 		this.id = Number(valueMap.get('Id'));
 		this.position = Number(valueMap.get('Pos'));
 		this.title = valueMap.get('Title');
+		this.name = valueMap.get('Name');
 		this.artist = valueMap.get('Artist');
 		this.album = valueMap.get('Album');
 		this.albumArtist = valueMap.get('AlbumArtist');
```
Regards

  Angus71

**Comment by hbenl** [comment]
Thanks! Interestingly I don't get a `Name` tag when fetching the playlist from mpd. What version of mpd are you using? Or are you using something else (like mopidy)?

**Comment by hbenl** [comment]
Anyway I've added the `name` property and published version 1.2.1 now.

**Comment by Angus71** [comment]
Thanks for the quick feedback and change ... The update works fine.

I'm using mpd 0.19.0 and created a playlist with Web Radio stations via the "mpc" commandline client (See example below). It looks like the 'Name' tag get's filled in by the "radio station".

```shell
mpc add http://streams.radiobob.de/bob-shlive/mp3-192/mediaplayer/
```
